packages/puppeteer/package.json

```diff
@@ -45,7 +45,7 @@
     "axe-core": "^4.0.1"
   },
   "peerDependencies": {
-    "puppeteer": "^1.10.0 || ^2.0.0"
+    "puppeteer": ">=1.10.0"
```

Comment:
This isn't safe. If Puppeteer v6 removes functionality we rely on, we'll be telling our users that this package works with v6 while it actually does not.

Reply:
I could do something like 
```json
	"puppeteer": ">=1.10.0 < 6"
```

Reply:
That seems reasonable to me